openmc/trigger.py

```diff
@@ -84,8 +84,36 @@ def get_trigger_xml(self, element):
 
         """
 
-        subelement = ET.SubElement(element, "trigger")
-        subelement.set("type", self._trigger_type)
-        subelement.set("threshold", str(self._threshold))
+        element = ET.Element("trigger")
+        element.set("type", self._trigger_type)
+        element.set("threshold", str(self._threshold))
         if len(self._scores) != 0:
-            subelement.set("scores", ' '.join(map(str, self._scores)))
+            element.set("scores", ' '.join(map(str, self._scores)))
```

Comment:
Pre-existing issue, but we make sure the `Trigger._scores` property is an `Iterable` of `str` already. Is mapping `str` onto the list redundant here?

```suggestion
            element.set("scores", ' '.join(self._scores))
```